src/System.Private.ServiceModel/tools/setupfiles/CleanupWCFTestService.cmd

```diff
@@ -1,20 +1,11 @@
 echo off
 setlocal
 
-echo BridgeKeepRunning=%BridgeKeepRunning%
-if '%BridgeKeepRunning%' neq 'true' (
-    echo Releasing Bridge resources and stopping it if running locally.
-    pushd %~dp0..\..\..\..\bin\wcf\tools\Bridge
-    echo Invoking Bridge.exe -stopIfLocal -reset %* ...
-    call Bridge.exe -stopIfLocal -reset %*
-    popd
-) else (
-    echo Releasing Bridge resources but leaving it running.
-    pushd %~dp0..\..\..\..\bin\wcf\tools\Bridge
-    echo Invoking Bridge.exe -reset %* ...
-    call Bridge.exe -reset %*
-    popd
-    echo The Bridge was left running because BridgeKeepRunning is true
-)
+echo Releasing Bridge resources and stopping it if running locally.
+pushd %~dp0..\..\..\..\bin\wcf\tools\Bridge
+echo Invoking Bridge.exe -stopIfAutoStart -reset %* ...
+call Bridge.exe -stopIfAutoStart -reset %*
```

Comment:
@Regarding @iamjasonp suggestion about start modes -- interesting and worth considering.  My main concern is that the max-idle-timeout is slightly orthogonal and can be applied to a Bridge started either manually or by a script.  Which brings us back to a 2-state value -- it was started by a script or it was started manually.  I rely on that max idle timeout to rescue CI machines that may have inadvertently not completed a test run cleanly.